src/gapi/gapiUtils.ml

```diff
@@ -77,6 +77,37 @@ let string_before_char c s =
     Str.string_before s first_occurrence
   with Not_found -> s
 
+let divide_string s c =
+  match String.index_opt s c with
+  | None -> "", s
+  | Some i ->
+    let len = String.length s in
+    let j = i + 1 in
+    String.sub s 0 i, String.sub s j (len - j)
+
+let strip_string s =
+  let leading_trailing_whitespace =
+    Str.regexp "\\(^[ \t\n]+\\|[ \t\n]+$\\)"
```

Comment:
I fixed a little bug here, adding `\r` to the regex.

Reply:
Thanks for catching that :)